src/main/java/cs2103/v15_1j/jimjim/command/SaveLocationCommand.java

```diff
@@ -21,7 +23,28 @@ public String getSavePath() {
     @Override
     public UIFeedback execute(ControllerStates conStates) {
     	String backup = conStates.config.savePath;
+    	File file = null;
+    	
+    	if (savePath.contains(" ")) {
```

Comment:
then if the user types in with a space we should add the `\` for them? if not it'll create two directories